### ISSUE TITLE
passing when it should fail

### ISSUE THREAD

**Issue by oprogramador** [import-js/eslint-plugin-import#2264]
I created this repo https://github.com/oprogramador/eslint-plugin-import-not-working .

The build with the lint command should fail because of the following:
```js
require('non-existent');
require('./non-existent');
```
in the `foo.js` file.

However, it's passing.
